.claude/scripts/plugin-definition-refresh.sh

```
@@ -0,0 +1,189 @@
+say() { [ "$QUIET" -eq 1 ] || printf '%s\n' "$*"; }
+
+while [ $# -gt 0 ]; do
+  case "$1" in
+    --repo-root) need $# "$1"; REPO_ROOT="$2"; shift 2 ;;
+    --plugins-root) need $# "$1"; PLUGINS_ROOT="$2"; shift 2 ;;
+    --gitlink) need $# "$1"; GITLINK="$2"; shift 2 ;;
+    --marketplace-dir) need $# "$1"; MARKETPLACE_DIR="$2"; shift 2 ;;
+    --marketplace) need $# "$1"; MARKETPLACE="$2"; shift 2 ;;
+    --plugin-id) need $# "$1"; PLUGIN_ID="$2"; shift 2 ;;
+    --submodule-path) need $# "$1"; SUBMODULE_PATH="$2"; shift 2 ;;
+    --installed) need $# "$1"; INSTALLED="$2"; shift 2 ;;
```

Shellcheck (0.11.0)

[warning] INSTALLED appears unused. Verify use (or export if used externally).

(SC2034)